java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3707,23 +3710,20 @@ public <ArgType> T xgroupSetId(
      *
      * @implNote ArgType is limited to String or GlideString, any other type will throw
      *     IllegalArgumentException
-     * @apiNote When in cluster mode, all keys in <code>keysAndIds</code> must map
-     * to the same hash slot.
-     * @see <a href="https://valkey.io/commands/xreadgroup/">valkey.io</a> for
-     *     details.
-     * @param keysAndIds A <code>Map</code> of keys and entry ids to read from.
-     *     The <code> Map</code> is composed of a stream's key and the id of the
-     *     entry after which the stream will be read. Use the special id of
-     *     <code>{@literal Map<String, Map<String, String[][]>>}
+     * @apiNote When in cluster mode, all keys in <code>keysAndIds</code> must map to the same hash
+     *     slot.
+     * @see <a href="https://valkey.io/commands/xreadgroup/">valkey.io</a> for details.
+     * @param keysAndIds A <code>Map</code> of keys and entry ids to read from. The <code> Map</code>
```

Comment:
nit
```suggestion
     * @param keysAndIds A <code>Map</code> of keys and entry ids to read from. The <code>Map</code>
```

Reply:
gonna revert comments on commands other than xinfo stream since they have been fixed in the transaction grooming PR